src/main/java/de/leonhard/storage/internal/DataStorage.java

```diff
@@ -319,7 +319,7 @@ default void setDefault(final String key, final Object value) {
       set(key, def);
       return def;
     } else {
-      return (T) ClassWrapper.getFromDef(raw, def);
+      return ClassWrapper.getFromDef(raw, def);
```

Comment:
What should this change do?

Reply:
It's a redundant cast, doesn't offer a whole lot other than making the code a bit nicer.

Reply:
But only the source code, the byte code still has this cast in it.